Frontend/ui-driver/src/Components/RideActionModal/View.purs

```diff
@@ -494,7 +495,7 @@ waitTimeView push config =
      , gravity START
      , orientation VERTICAL
      , weight 1.0
-     , visibility if (config.waitTime /= "__") && (config.notifiedCustomer) && ((getValueToLocalStore IS_WAIT_TIMER_STOP) /= "NoView" ) && ((getValueToLocalStore IS_WAIT_TIMER_STOP) /= "Stop" ) then VISIBLE else GONE
+     , visibility if (config.waitTime /= "__") && (config.notifiedCustomer) && (getValueToLocalStore WAITING_TIME_STATUS) == (show ST.PostTriggered) then VISIBLE else GONE
```

Comment:
can you maintain  `WAITING_TIME_STATUS` in local state also. Because directly accessing the local store will not work properly.

Reply:
@Vignesh-772  we tried it
It doesn't make any difference
CC @witcher-shailesh